goody-server-api/src/main/java/org/re/goody/chat/service/ChatInquiryService.java

```diff
@@ -140,6 +135,13 @@ public String changeInquiryStatus(User user, Long inquiryId, InquiryStatusReques
 				inquiryDomainService.changeInquiryStatusToCompleted(user.getId(), inquiry);
 				userService.increaseCompletedInquiryCount(
 					List.of(inquiry.getPostOwner().getId(), inquiry.getInquirer().getId()));
+
+				Long postId = inquiry.getPost().getId();
+				if (inquiryDomainService.getActiveInquiryCountByPostOwnerId(inquiry.getPostOwner().getId(), postId) <= 0
```

Comment:
```suggestion
				if (!hasActiveInquiry(postId)
```

진행중인 거래의 개수를 다 카운트 할 필요는 없지 않을까요? 
진행중인 거래의 유무로만 판단해도 괜찮을 것 같습니다!

Reply:
진행중인 거래 유무로만 판단하도록 수정하였습니다~